docs/root/intro/version_history.rst

```diff
@@ -3,6 +3,7 @@ Version history
 
 1.11.0 (Pending)
 ================
+* access log: added RESPONSE_CODE_DETAILS.
```

Comment:
nit: link to the doc (as well as gRPC field)

Reply:
Done